lib/plots/canopy.js

```diff
@@ -196,11 +196,6 @@ export class Pannable extends Etch {
     </div>;
   }
 
-  readAfterUpdate () {
-    this.containerRect =
-      this.element.getElementsByClassName('ink-pannable')[0].getBoundingClientRect()
-  }
-
```

Comment:
removed this definition since the new one has been introduced:

https://github.com/JunoLab/atom-ink/blob/58320aec36de40e67832669b03ba5d1586bb392c/lib/plots/canopy.js#L98-L101

Reply:
I don't think that's correct -- we need to merge the two methods. Will do that in a separate PR though.

Reply:
okay, I just reverted the part and left a TODO